pyproject.toml

```diff
@@ -18,13 +18,20 @@ include = ["schema-am.rng", "install/*", "lnst-ctl.conf"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-
-pyroute2 = "*"
-pyyaml = "*"
```

Comment:
`pyyaml` is used by `TRexLib`

https://github.com/LNST-project/lnst/blob/60bdddf3f4f5970809ca4fc9c97cf9d444f6c9b5/lnst/External/TRex/TRexLib.py#L9

Reply:
Thanks, as `pyyaml` is used in `trex` recipes only, i've moved it to the `trex` extra group